commands/credentials/repair_default.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+
+function * run (context, heroku) {
+  const fetcher = require('../../lib/fetcher')(heroku)
+  const host = require('../../lib/host')
+  const util = require('../../lib/util')
+
+  const {app, args} = context
+
+  let db = yield fetcher.addon(app, args.database)
+  if (util.starterPlan(db)) throw new Error('This operation is not supported by Hobby tier databases.')
```

Comment:
I wonder if we should explain why? I.e., that they don't have configurable permissions in the first place... it's fine for now, though; we can address that during the beta if there's confusion...